tests/conftest.py

```diff
@@ -1,12 +1,18 @@
 import functools
+from typing import Any, Callable, Dict
 
 import pytest
 
 from starlette.testclient import TestClient
 
+TestClientFactory = Callable[..., TestClient]
+
 
 @pytest.fixture
-def test_client_factory(anyio_backend_name, anyio_backend_options):
+def test_client_factory(
+    anyio_backend_name: str,
+    anyio_backend_options: Dict[str, Any],
```

Comment:
Is the type available in AnyIO?

Reply:
As far as I can see, it looks like it uses the same str and Dict[str, Any] in AnyIO too.

Example [here](https://github.com/agronholm/anyio/blob/137de708f40262e959d9d43178dda600432da56b/tests/test_debugging.py#L29)